### PR TITLE
fix: hardcode the transaction fee

### PR DESCRIPTION
**Pull request by rishkwal**
NOTE: This is a temporary fix.
This fees should be decided by the maker.
This hardcodes the fee at the taker

### CODE REVIEW

File: src/taker/api.rs [hunk 1]
```diff
@@ -71,6 +71,8 @@ pub const RECONNECT_SHORT_SLEEP_DELAY: u64 = 10;
 pub const RECONNECT_LONG_SLEEP_DELAY: u64 = 60;
 pub const SHORT_LONG_SLEEP_DELAY_TRANSITION: u32 = 60;
 pub const RECONNECT_ATTEMPT_TIMEOUT_SEC: u64 = 300;
+// TODO: Maker should decide this fee_rate
```
Comment (mojoX911) [inline src/taker/api.rs:74]: This is not actually a feerate but a total. fee. Change the variable name to just `MINER_FEE`. Also add a doc saying that this fee is used for both funding and contract transactions.

**Review by mojoX911 — COMMENTED** [review]
Ack. Please open up an issue for making this configurable at maker's. And mark it for v0.1.1. 

One nit.

File: src/taker/api.rs (hunk 1 above)
Comment (rishkwal) [reply src/taker/api.rs]: Done

**Review by mojoX911 — APPROVED** [review]
Ack

**Pull request merged**